cookbooks/aws-parallelcluster-install/recipes/sudoers.rb

```diff
@@ -4,21 +4,26 @@
 # Cookbook:: aws-parallelcluster
 # Recipe:: sudoers
```

Comment:
Nitpick: shouldn't we change the recipe name to `sudo` since it includes `sudo` package installation?

Reply:
I'm not sure about this, the sudo package is by default installed in most of the OSes, we're just installing it to be sure the next step work (the sudo installation is more a prerequisites for the following step). Let's talk about this